### PR TITLE
Single bigip tests

### LINKED ISSUE #552: singlebigip tests a subset of the original functional tests, do not run in the nightly regression
When those tests run in nightly this issue is resolved.

### PR DESCRIPTION
**Pull request by zancas**
Fixes #552 
@richbrowne 

#### What issues does this address?
This PR adds a subset of the "legacy tests" to the nightly regression runs.

#### What's this change do?
The legacy tests are reorganized by their infrastructure requirements.  This PR adds the tests that only require a singelbigip.

### CODE REVIEW

**Comment by zancas** [comment]
PLEASE MERGE #527 first!

**Comment by richbrowne** [comment]
Can we change the package dependencies to sdk version 2.1 and keep the 2.2 dependency in the requirements.txt file?

File: .containenv/Dockerfile.ubuntu [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM        ubuntu:latest
```
Comment (richbrowne) [inline .containenv/Dockerfile.ubuntu]: Save this for another commit.

File: tox.ini [hunk 2]
```diff
@@ -1,29 +1,18 @@
-[tox]
```
Comment (richbrowne) [inline tox.ini]: Why was this removed?

File: tox.ini [hunk 3]
```diff
@@ -1,29 +1,18 @@
...
-envlist =
```
Comment (richbrowne) [inline tox.ini]: Ditto.  Do we not need this?

File: tox.ini [hunk 4]
```diff
@@ -1,29 +1,18 @@
...
-    unit
-    unit-buildbot
-    functest
-    functest-buildbot
-
 [testenv:style]
-basepython = python2.7
```
Comment (richbrowne) [inline tox.ini]: why remove basepython?

File: requirements.functest.txt [hunk 5]
```diff
@@ -10,12 +8,11 @@ git+https://github.com/openstack/neutron-lbaas.git@stable/liberty
...
 
 # Test utilities
-git+https://github.com/F5Networks/f5-openstack-test.git
 git+https://github.com/F5Networks/pytest-symbols.git
 
 mock==2.0.0
-pytest==3.0.3
-pytest-cov==2.3.1
+pytest>=3.0.5,<4
+pytest-cov==2.4.0,<3
```
Comment (richbrowne) [inline requirements.functest.txt]: If it's equal to 2.4.0, do we need to say <3?

File: requirements.functest.txt [hunk 6]
```diff
@@ -10,12 +8,11 @@ git+https://github.com/openstack/neutron-lbaas.git@stable/liberty
 git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver.git@liberty
 
 # Test utilities
-git+https://github.com/F5Networks/f5-openstack-test.git
 git+https://github.com/F5Networks/pytest-symbols.git
 
 mock==2.0.0
-pytest==3.0.3
-pytest-cov==2.3.1
+pytest>=3.0.5,<4
```
Comment (richbrowne) [inline requirements.functest.txt]: Do we really need to update these requirements?  Are they giving us something we need?

File: tox.ini [hunk 7]
```diff
@@ -1,29 +1,18 @@
...
+deps = -r{toxinidir}/requirements.unittest.txt
 changedir = f5_openstack_agent
 commands = py.test -svvra --cov {posargs}
 
-[testenv:functest]
-basepython = python2.7
+[testenv:disconnected_service]
 changedir = test/functional/
-deps = -rrequirements.functest.txt
-commands = py.test --cov -svvra {posargs}
```
Comment (richbrowne) [inline tox.ini]: Why remove the -svvra?  I like having that output.
